tests/integration/test_vectordraw.py

```diff
@@ -0,0 +1,1032 @@
+            # Find tip
+            board_has_tip = self.board_has_point(vector["expected_tip_position"], point_elements)
+            # Find label
+            board_has_label = self.board_has_label(board, vector["name"])
+            # Check if line, tip, tail are present
+            if vector["render"]:
+                self.assertTrue(board_has_line)
+                self.assertTrue(board_has_tail)
+                self.assertTrue(board_has_tip)
+                self.assertTrue(board_has_label)
+            else:
+                self.assertFalse(board_has_line)
+                self.assertFalse(board_has_tail)
+                self.assertFalse(board_has_tip)
+                self.assertFalse(board_has_label)
+
+    def assert_points(self, board, points):
+        point_elements = board.find_elements_by_css_selector("ellipse")
+        for point in points:
+            board_has_point = self.board_has_point(point["expected_position"], point_elements)
+            self.assertEquals(board_has_point, point["render"])
+
+    def board_has_line(self, position, line_elements):
+        line = self.find_line(position, line_elements)
+        return bool(line) and self.line_has_title(line) and self.line_has_desc(line)
+
+    def board_has_point(self, position, point_elements):
+        return bool(self.find_point(position, point_elements))
+
+    def board_has_label(self, board, label_text):
+        text_elements = board.find_elements_by_css_selector(".JXGtext")
+        for text_element in text_elements:
+            is_tick = "ticks" in text_element.get_attribute("id")
+            if not is_tick and text_element.text == label_text:
+                return True
+        return False
+
+    def line_has_title(self, line):
+        title = line.find_element_by_css_selector("title")
+        title_id = title.get_attribute("id")
+        aria_labelledby = line.get_attribute("aria-labelledby")
+        return title_id == aria_labelledby
+
+    def line_has_desc(self, line):
+        aria_describedby = line.get_attribute("aria-describedby")
+        return aria_describedby == "jxgboard1-vector-properties"
+
+    def find_line(self, position, line_elements):
+        expected_line_position = position.items()
+        for line in line_elements:
+            line_position = {
+                "x1": int(float(line.get_attribute("x1"))),
+                "y1": int(float(line.get_attribute("y1"))),
+                "x2": int(float(line.get_attribute("x2"))),
+                "y2": int(float(line.get_attribute("y2"))),
+            }.items()
+            if line_position == expected_line_position:
+                return line
+
+    def find_point(self, position, point_elements):
+        expected_position = position.items()
+        for point in point_elements:
+            point_position = {
+                "cx": int(round(float(point.get_attribute("cx")))),
```

Comment:
Same comment here. If you want to just get an `int` and discard everything after the decimal point, you can do this:
```python
int(line.get_attribute("x1").split(".", 1)[0])
```